### ISSUE TITLE
[Recommendation]

### ISSUE THREAD

**Issue by ChrisLopez-code** [eborello/woop#64]
I think you guys are using the name of the show to block a spoiler, you should probably change it so that you won't require the name of the show to properly function. Example, if I looked up "Chuck kills Jimmy" in reference to better call saul, there is no mention of the show, just the characters.

**Comment by JeffPerando** [comment]
Yeah, they should use a set of predefined keywords to look for, as well as main characters from the show. So things like "spoiler", "ending", "death", should be looked for, in addition to the show info itself.

**Comment by JeffPerando** [comment]
Thinking more about it, they could use the tags a YT video has associated with it, as well as the media it features, and hashtags.
